### DISCUSSION TITLE
Ability to name a deploy from sf project deploy

### DISCUSSION THREAD

**Discussion by davidCburke** [forcedotcom/cli#2722] [Ideas]
It would be handy to be able to give a deploy a name so when you look at the deploy status in Salesforce you don't just see the deploy ID

**Comment by mshanemc** [comment]
This is the API that we're working with. https://developer.salesforce.com/docs/atlas.en-us.api_meta.meta/api_meta/meta_deploy.htm

That would need to change to make names possible, and then additional work to display the name
